lib/plugins/aws/package/lib/getHashForFilePath.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const memoize = require('memoizee');
+const crypto = require('crypto');
+const fs = require('fs');
+
+const getHashForFilePath = memoize(async filePath => {
+  const fileHash = crypto.createHash('sha256');
+  fileHash.setEncoding('base64');
+  try {
+    fileHash.write(await fs.promises.readFile(filePath));
```

Comment:
Let's calculate hash without loading whole file contents into process memory.

You can steal logic that relies on stream read from `addFileContentsToHashes`